SECURITY.md

```diff
@@ -0,0 +1,11 @@
+# Security Policy
+
+## Supported Versions
+
+Please see [Releases](https://github.com/prysmaticlabs/prysm/releases/). We recommend using the [most recently released version](https://github.com/prysmaticlabs/prysm/releases/latest).
+
+## Reporting a Vulnerability
+
+Please send vulnerability reports to security@chainsafe.io.
```

Comment:
This doesn't seem right :smile: 

```suggestion
Please send vulnerability reports to security@prysmaticlabs.com.
```

Reply:
Very true, I commited a few other ones and got it mixed up! :)